Classes/Controller/EntityFactController.php

```diff
@@ -73,7 +73,7 @@ public function showAction()
             $viewArray = $apiAnswerDecode;
         }
 
-        $viewArray['describedBy'] = $apiAnswerDecode['describedBy'];
+        $viewArray["describedBy"] = $apiAnswerDecode["describedBy"];
```

Comment:
Any reason for this change?

Please always use single quotes for array keys and strings. Double quotes are only used to add the newline character `\n` to a string.

See: https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/CodingGuidelines/CglPhp/PhpFileFormatting/PhpSyntaxFormatting.html#strings

Reply:
Once got a php error that wanted that change. I can't replicate the error now, so I'll happily change it back.